vertical-pod-autoscaler/pkg/updater/eviction/pods_eviction_restriction_test.go

```diff
@@ -223,6 +240,32 @@ func TestEvictReplicatedByController(t *testing.T) {
 				},
 			},
 		},
+		{
+			// Cannot evict a single Pod under default settings.
```

Comment:
Even better would be to add `name` string to test case and then use that instead of index in the loop. Then:

- It works as a comment
- Makes it easy to find which test case failed

Reply:
OK, I moved newly added & pre-existing comments to TC name.